package.json

```diff
@@ -30,11 +30,11 @@
   "homepage": "https://github.com/indiana-university/rivet-icons#readme",
   "dependencies": {
     "fast-glob": "3.2.12",
-    "svg-sprite": "2.0.0"
+    "svg-sprite": "2.0.2"
   },
   "devDependencies": {
-    "@11ty/eleventy": "1.0.2",
-    "gh-pages": "4.0.0",
+    "@11ty/eleventy": "2.0.1",
+    "gh-pages": "5.0.0",
```

Comment:
Remove the `gh-pages` dependency and the `deploy` and `gh-pages` npm scripts.

Reply:
Make sure to also do `npm install` after removing the `gh-pages` dependency, so that `package-lock.json` gets updated.